docs/UserGuide.md

```diff
@@ -1,186 +1,239 @@
+### 2.3. Listing the tasks: `list`
 
-Examples:
+Shows a list of all the tasks in the task manager.<br>
+> Listing is done automatically when user switches between the tabs.<br>
+> However, user can specify to list all the tasks he has in, which will be shown in a pop up.<br>
+> User can also sort it accordingly to the deadlines, priorities, etc.
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+Shows a list of all the tasks in lexographical order.<br>
+Format: `list all`
 
-### 2.3. Listing all persons : `list`
+Shows a list of all the tasks sorted by deadlines.<br>
+Format: `list deadlines`
 
-Shows a list of all persons in the address book.<br>
-Format: `list`
+Shows a list of all the tasks sorted by priority.<br>
+Format: `list priority`
 
-### 2.4. Editing a person : `edit`
+Shows a list of the tasks sorted by a stated priority level, from 1 - 4.<br>
+Format: `list priority PRIORITY_LEVEL`
+> Tasks can be given any priority level from 0 to 3.
+> Tasks with PRORITY_LEVEL priority will be displayed.
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Examples:
+* `list priority 1`
+* `list priority 2`
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
-    The index **must be a positive integer** 1, 2, 3, ...
-> * At least one of the optional fields must be provided.
-> * Existing values will be updated to the input values.
-> * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+Shows a list of tasks of the tags sorted in lexographical order.<br>
+Format: `list t/TAGS...`
+> The tasks listed will be in clusters according to tags, but sorted in alphabetical order.<br>
 
 Examples:
+* `list t/CS3230 t/Work`
 
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
+### 2.4. Editing an existing task: `edit`
+Edits an existing task in the task manager.<br>
+Format: `edit i/INDEX [n/TASK_NAME] [d/DEADLINE] [p/PRIORITY_LEVEL] [i/ANY_INFO] [t/TAGS]...`
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+> * Edits the task labeled by the INDEX digit as shown on the screen. The INDEX must be a positive integer, e.g. 1, 2, 3, ...
+> * At least one of the optional [fields] must be provided.
+> * Existing fields will be overwritten.
 
-### 2.5. Finding all persons containing any keyword in their name: `find`
+Examples:
+* `edit i/1 n/Assignment 2 d/25 Feb 2018 p/2`
+* `edit i/4 n/Exercise`
+
+### 2.5. Shifting tabs: `switch`
```

Comment:
Explaining `switch` here seems rather abrupt.  May I suggest to moving it to the front, even before `add`?